### PR TITLE
JESI-2484

### PR DESCRIPTION
**Pull request by axrs**
[JESI-2484](https://jesims.atlassian.net/browse/JESI-2484)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -111,11 +113,13 @@ RUN npm install --global --unsafe-perm \
...
-RUN pip3 install --upgrade pip && pip3 install --upgrade \
-	PyYAML==3.10 \
-	awscli \
-	awsebcli \
-	&& rm -rf ~/.cache/pip
+RUN pip3 install --upgrade pip setuptools && pip3 --no-cache-dir install \
+	'PyYAML<=3.13,>=3.10' \
+	'botocore<1.13,>=1.12.29' \
+	'colorama<0.4.0,>=0.3.9' \
+	'urllib3<1.25,>=1.24.1' \
```
Comment (axrs) [inline Dockerfile]: fing hate python

File: Dockerfile [hunk 2]
```diff
@@ -126,6 +130,14 @@ RUN git clone -b master https://github.com/jesims/circleci-tools.git \
 ENV PATH=$PATH:/tmp/circleci-tools/
 RUN node -v > .node_version
 
+#-- Install cljog
+RUN wget https://raw.githubusercontent.com/axrs/cljog/${CLJOG_VERSION}/cljog \
+  && chmod u+x cljog \
+  && mv cljog /usr/bin/ \
+  && wget https://raw.githubusercontent.com/axrs/cljog/${CLJOG_VERSION}/example-scripts/echo.clj \
+  && chmod u+x echo.clj \
+  && ./echo.clj
```
Comment (AndreTheHunter) [inline Dockerfile]: then remove ./echo.clj
```suggestion
  && ./echo.clj && rm echo.clj
```

File: Dockerfile [hunk 3]
```diff
@@ -126,6 +130,14 @@ RUN git clone -b master https://github.com/jesims/circleci-tools.git \
 ENV PATH=$PATH:/tmp/circleci-tools/
 RUN node -v > .node_version
 
+#-- Install cljog
+RUN wget https://raw.githubusercontent.com/axrs/cljog/${CLJOG_VERSION}/cljog \
+  && chmod u+x cljog \
+  && mv cljog /usr/bin/ \
```
Comment (AndreTheHunter) [inline Dockerfile]: Not `/usr/local/bin`?
"/usr/local/bin is for normal user programs not managed by the distribution package manager"
```suggestion
  && mv cljog /usr/local/bin/ \
```

Comment (axrs) [reply Dockerfile]: 👍

File: Dockerfile (hunk 2 above)
Comment (axrs) [reply Dockerfile]: 👍

**Pull request merged**